webview/src/setup/components/experiments/Experiments.tsx

```diff
@@ -42,16 +42,16 @@ export const Experiments: React.FC<ExperimentsProps> = ({ isDvcSetup }) => {
   return (
     <EmptyState isFullScreen={false}>
       <h1>Setup Complete</h1>
-      <p>
-        Learn more about using DVCLive in the{' '}
-        <a href="https://dvc.org/doc/dvclive">docs</a>.
-      </p>
       <IconButton
         appearance="primary"
         icon={Beaker}
         onClick={showExperiments}
         text="Show Experiments"
       />
+      <p>
+        Learn more about using DVCLive in the{' '}
```

Comment:
[F] I don't think it made much sense to have the button between this line and the actual examples.